### ISSUE TITLE
Node doesn't start in simple_add_new_node_while_schema_changes_test

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylla#6011]
Most recently seen in https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/418/testReport/update_cluster_layout_tests/TestUpdateClusterLayout/simple_add_new_node_while_schema_changes_test/
but the regression appears to have been introduced before repair-based operations (in https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/410/testReport/update_cluster_layout_tests/TestUpdateClusterLayout/simple_add_new_node_while_schema_changes_test/ or earlier)
```
Traceback (most recent call last):
  File "/usr/lib64/python3.7/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/usr/lib64/python3.7/unittest/case.py", line 645, in run
    testMethod()
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/update_cluster_layout_tests.py", line 605, in simple_add_new_node_while_schema_changes_test
    node4.watch_log_for("Beginning stream session")
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-ccm/ccmlib/node.py", line 448, in watch_log_for
    [e.pattern for e in tofind]) + ":\n" + reads[:50] + ".....\nSee {} for remainder".format(filename))
ccmlib.node.TimeoutError: 02 Mar 2020 00:48:06 [node4] Missing: ['Beginning stream session']:
```

https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/418/artifact/logs-release.2/1583931622066_update_cluster_layout_tests.TestUpdateClusterLayout.simple_add_new_node_while_schema_changes_test/node4.log
```
INFO  2020-03-11 12:50:13,347 [shard 1] repair - repair id 1 on shard 1 failed: std::_Nested_exception<no_such_column_family> (Can't find a column family cf in keyspace ks): std::out_of_range ()
...
INFO  2020-03-11 12:50:13,521 [shard 0] repair - repair id 1 on shard 0 failed: std::_Nested_exception<no_such_column_family> (Can't find a column family cf in keyspace ks): std::out_of_range ()
INFO  2020-03-11 12:50:13,521 [shard 0] repair - repair id 1 failed: std::runtime_error ({shard 0: std::_Nested_exception<no_such_column_family> (Can't find a column family cf in keyspace ks): std::out_of_range (), shard 1: std::_Nested_exception<no_such_column_family> (Can't find a column family cf in keyspace ks): std::out_of_range ()})
INFO  2020-03-11 12:50:13,521 [shard 0] repair - repair id 1 to sync data for keyspace=ks, status=failed: std::runtime_error ({shard 0: std::_Nested_exception<no_such_column_family> (Can't find a column family cf in keyspace ks): std::out_of_range (), shard 1: std::_Nested_exception<no_such_column_family> (Can't find a column family cf in keyspace ks): std::out_of_range ()})
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down gossiping
INFO  2020-03-11 12:50:13,522 [shard 0] gossip - My status = BOOT
WARN  2020-03-11 12:50:13,522 [shard 0] gossip - No local state or state is in silent shutdown, not announcing shutdown
INFO  2020-03-11 12:50:13,522 [shard 0] gossip - Disable and wait for gossip loop started
INFO  2020-03-11 12:50:13,522 [shard 0] gossip - Disable and wait for gossip loop finished
INFO  2020-03-11 12:50:13,522 [shard 0] gossip - Gossip is now stopped
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down gossiping was successful
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down storage service notifications
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down storage service notifications was successful
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down repair service
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down repair service was successful
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down cdc
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down cdc was successful
INFO  2020-03-11 12:50:13,522 [shard 0] init - Shutting down migration manager
INFO  2020-03-11 12:50:13,522 [shard 0] migration_manager - stopping migration service
INFO  2020-03-11 12:50:13,523 [shard 1] migration_manager - stopping migration service
INFO  2020-03-11 12:50:13,523 [shard 0] init - Shutting down migration manager was successful
INFO  2020-03-11 12:50:13,523 [shard 0] init - Shutting down database
INFO  2020-03-11 12:50:13,523 [shard 0] compaction_manager - Asked to stop
INFO  2020-03-11 12:50:13,523 [shard 0] compaction_manager - Stopped
INFO  2020-03-11 12:50:13,523 [shard 1] compaction_manager - Asked to stop
INFO  2020-03-11 12:50:13,523 [shard 1] compaction_manager - Stopped
INFO  2020-03-11 12:50:13,542 [shard 0] init - Shutting down database: waiting for background jobs...
INFO  2020-03-11 12:50:13,542 [shard 0] init - Shutting down database was successful
INFO  2020-03-11 12:50:13,542 [shard 0] init - Shutting down migration manager notifier
INFO  2020-03-11 12:50:13,542 [shard 0] init - Shutting down migration manager notifier was successful
INFO  2020-03-11 12:50:13,542 [shard 0] init - Shutting down token metadata
INFO  2020-03-11 12:50:13,543 [shard 0] init - Shutting down token metadata was successful
INFO  2020-03-11 12:50:13,543 [shard 0] init - Shutting down prometheus API server
INFO  2020-03-11 12:50:13,544 [shard 0] init - Shutting down prometheus API server was successful
INFO  2020-03-11 12:50:13,544 [shard 0] init - Shutting down sighup
INFO  2020-03-11 12:50:13,544 [shard 0] init - Shutting down sighup was successful
ERROR 2020-03-11 12:50:13,544 [shard 0] init - Startup failed: std::runtime_error ({shard 0: std::_Nested_exception<no_such_column_family> (Can't find a column family cf in keyspace ks): std::out_of_range (), shard 1: std::_Nested_exception<no_such_column_family> (Can't find a column family cf in keyspace ks): std::out_of_range ()})
```

**Comment by bhalevy** [comment]
@asias please look into this ad determine if the problem is with scylla or with the test.

**Comment by slivne** [comment]
@asias I think we changed repair to synch schema before starting - why is this failing ?

**Comment by asias** [comment]
node4.log shows the test is using scylla d895f5e1311, which does not contain my fix https://github.com/scylladb/scylla/pull/5986. Let's retest when the fix is merged.  5986 should fix the issue here too.

```
INFO  2020-03-11 12:48:43,556 [shard 0] init - Scylla version 666.development-0.20200310.d895f5e1311 with build-id 5278b18aaf99be63b06c7c4d515c9a14822d3692 starting ...
```

**Comment by asias** [comment]
Duplication of #5942

**Comment by asias** [comment]
> @asias I think we changed repair to synch schema before starting - why is this failing ?

Because keyspace is dropped in the middle of repair.

**Comment by bhalevy** [comment]
Duplicate of #5942
